server/src-lib/Hasura/RQL/DDL/EventTrigger.hs

```diff
@@ -354,3 +374,12 @@ updateEventTriggerInCatalog trigConf =
       configuration = $1
       WHERE name = $2
     |] (Q.AltJ $ toJSON trigConf, etcName trigConf) True
+
+updatePauseOfEventTriggerInCatalog :: (MonadTx m) => TriggerName -> Bool -> m Int
+updatePauseOfEventTriggerInCatalog triggerName pause = liftTx $ do
```

Comment:
This function name is a little unwieldy; I’d rename it to something a little more direct:

```suggestion
setEventTriggerPausedInCatalog :: (MonadTx m) => TriggerName -> Bool -> m Int
setEventTriggerPausedInCatalog triggerName pause = liftTx $ do
```